docs/data/material/components/container/container.md

```diff
@@ -35,10 +35,10 @@ The max-width matches the min-width of the current breakpoint.
 <Container fixed>
 ```
 
-## Experimental APIs
+## Toolpad
 
 ### Page Container
 
-The [PageContainer](https://mui.com/toolpad/core/react-page-container/) component in `@toolpad/core` is the ideal wrapper for the content of your dashboard. It makes the Material UI Container navigation aware and extends it with page title, breadcrumbs, actions, and more.
+The [PageContainer](https://mui.com/toolpad/core/react-page-container/) component in `@toolpad/core` is the ideal wrapper for the content of your dashboard. It makes the Material UI Container navigation-aware and extends it with page title, breadcrumbs, actions, and more.
 
 {{"demo": "../breadcrumbs/PageContainerBasic.js", "height": 400, "hideToolbar": true}}
```

Comment:
The dark/light mode is broken on all those demos.

As far as I understand things, `theme={demoTheme}` shouldn't be provided, not be the `AppProvider` responsibility to have this. Imagine I already have an application configured with a theme, I want some of my pages to have a Toolpad container, this shouldn't break. I don't buy that https://marmelab.com/react-admin/Theming.html is right.

At minimum, MUI System should support theme nesting where you can inherit the light/dark mode and use different values for the other values.

Reply:
`const demoWindow = window !== undefined ? window() : undefined;` feels confusing, I think it should copy https://github.com/mui/material-ui/blob/a80eb394b7cf2dae94caebe05dd514906a1ea426/docs/data/material/components/app-bar/HideAppBar.js#L31

much clearer.